services/pull/check.go

```diff
@@ -36,6 +36,7 @@ var (
 	ErrUserNotAllowedToMerge = errors.New("user not allowed to merge")
 	ErrHasMerged             = errors.New("has already been merged")
 	ErrIsWorkInProgress      = errors.New("work in progress PRs cannot be merged")
+	ErrIsChecking            = errors.New("PRs which are still being conflict checked cannot be merged")
```

Comment:
Alternatively,
```suggestion
	ErrIsChecking            = errors.New("cannot merge while conflict checking is in progress")
```
I'm fine with both.